doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -25,165 +25,51 @@ Some Key Differences
 
 .. list-table::
 
-   * - In MATLAB®, the basic data type is a multidimensional array of
-       double precision floating point numbers.  Most expressions take such
-       arrays and return such arrays.  Operations on the 2-D instances of
-       these arrays are designed to act more or less like matrix operations
-       in linear algebra.
-     - In NumPy the basic type is a multidimensional ``array``.  Operations
-       on these arrays in all dimensionalities including 2D are element-wise
-       operations.  One needs to use specific functions for linear algebra
-       (though for matrix multiplication, one can use the ``@`` operator
-       in python 3.5 and above).
-
-   * - MATLAB® uses 1 (one) based indexing. The initial element of a
+   * - In MATLAB, the basic type is a multidimensional array. By default, MATLAB
```

Comment:
```suggestion
   * - In MATLAB, the basic nonscalar type is a multidimensional array. By default, MATLAB
```

Reply:
Matlab defaults to a 2D array even for scalar assignments. So when you say `a=1`, it creates a 1x1 array and assigns it to `a`. You can reference the `1` by using `a(1)` or `a(1,1)`. 

I don't think matlab has a "scalar" assignment option because all variables are given dimensions. It does consider a 1x1 matrix a scalar, so

```matlab
>> a=1
>> isscalar(a)
TF =
    1
>> isscalar(a(1,1))
TF =
    1
```

This is kind of a strange convention in matlab.

Reply:
Interesting! 
```
- In MATLAB, the basic type, even for scalars, is a multidimensional array.
```
Ordinarily I wouldn't tell MATLAB users what they already know, but this behavior is going to be contrasted with NumPy's.

Reply:
I like that change, @bjnath . 

I'll add the `even for scalars`. MATLAB's choice of everything is a matrix makes some things easier, but if you're trying to convert your code to NumPy, there are a few hiccups like this that come up. 

I don't think most MATLAB users think about the difference between scalar/vector/matrix in normal operations because it all works out seemlessly.